src/style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Alkatra', cursive;
+}
+
+body {
+  font-size: 16px;
+  background-color: rgb(216, 227, 236);
+}
+
+ul {
+  list-style: none;
+}
+
+button {
+  cursor: pointer;
+  border: none;
+}
+
+input {
+  outline: none;
+  border: none;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#container {
+  max-width: 400px;
+  width: 100%;
+  height: auto;
+  background-color: #fff;
+  border: 1px solid #989898;
+
+  /* box-shadow: 5px 5px 5px #000000; */
```

Comment:
[OPTIONAL] Please don't leave any uncommented code in your codebase. In this way, you can keep your codebase clean, neat, and professional. If you need a specific line of code, you can use git history.